vaultpress.php

```diff
@@ -14,6 +14,32 @@
 // don't call the file directly
 defined( 'ABSPATH' ) or die();
 
+/**
+ * Load all the packages.
+ *
+ * We want to fail gracefully if `composer install` has not been executed yet, so we are checking for the autoloader.
+ * If the autoloader is not present, let's log the failure, pause VaultPress, and display a nice admin notice.
+ */
+$jetpack_autoloader = dirname( __FILE__ ) . '/vendor/autoload.php';
```

Comment:
this is not a "jetpack_autoloader". It's just an autoloader. You can call this var `$loader` or `$autoloader` if you like.

Reply:
Thanks for this! I was planning to clean that up, but never did 😅 (Miguel fixed it in https://github.com/Automattic/vaultpress/pull/5/commits/76c46134dfd7453b49fc0cf2a2641ec332c8526d)